### PR TITLE
🎨 Update app view for better alignment to updated design

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?

- Update UI in the app view to better align the design with the new designs
- Small UI improvements to buttons and dropdown menu to make them easier to read

## Related issue/s
fixes https://github.com/ITISFoundation/osparc-issues/issues/1223

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/auth/LoginPageS4L.js [hunk 1]
```diff
@@ -52,12 +52,8 @@ qx.Class.define("osparc.auth.LoginPageS4L", {
 
     __setBackgroundImage: function() {
       let backgroundImage = "";
-      const today = new Date();
-      const xmas = new Date(2023, 12, 24);
-      const endDay = new Date(new Date().setDate(xmas.getDate() + 12));
-      const isXmasDays = today < endDay;
 
-      const defaultBG = isXmasDays ? "url(resource/osparc/Santa_Billie.png)" : "url(resource/osparc/Sim4Life_login_page_master_transparent_bg.png)";
+      const defaultBG = "url(resource/osparc/Sim4Life_login_page_master_transparent_bg.png)";
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/auth/LoginPageS4L.js:56]: Q: sorry, here a recurrent question: is there a way to put these assets in https://github.com/ZurichMedTech/s4l-assets instead?

**Review by pcrespov — COMMENTED** [review]
:nail_care:

**Review by sanderegg — APPROVED** [review]

**Review by matusdrobuliak66 — COMMENTED** [review]
:+1:

**Review by elisabettai — APPROVED** [review]
Looking very good, thanks!

Maybe you've noticed already: something went a bit wrong with one of the buttons in the PO center: https://github.com/ITISFoundation/osparc-issues/issues/1223. It's not critical, but maybe easy to squeeze it in this PR.

**Review by ignapas — APPROVED** [review]
lgtm

File: services/static-webserver/client/source/class/osparc/auth/LoginPageS4L.js (hunk 1 above)
Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/auth/LoginPageS4L.js:56]: this image is being removed. It's not the best idea to have internal product assets outside your repo as some one might delete not knowing there is a dependency on it. I can add the image there as a back up?

**Comment by jsaq007** [comment]
> Looking very good, thanks!
> 
> Maybe you've noticed already: something went a bit wrong with one of the buttons in the PO center: [ITISFoundation/osparc-issues#1223](https://github.com/ITISFoundation/osparc-issues/issues/1223). It's not critical, but maybe easy to squeeze it in this PR.

Yes this should fix at least that and a couple other things I saw there

**Review by SCA-ZMT — APPROVED** [review]

**Pull request merged**